bacs/src/main/java/com/adyen/checkout/bacs/BacsDirectDebitComponentParamsMapper.kt

```diff
@@ -31,7 +32,7 @@ internal class BacsDirectDebitComponentParamsMapper(
             shopperLocale = parentConfiguration.shopperLocale,
             environment = parentConfiguration.environment,
             clientKey = parentConfiguration.clientKey,
-            amount = bacsDirectDebitConfiguration.amount,
+            amount = bacsDirectDebitConfiguration.amount ?: Amount.EMPTY,
```

Comment:
Do we have tests that validate the default values? If tests validate this, then you can't accidentally change a default value

Reply:
Just wanted to make sure ✅